server/service/src/sync/translations/stock_line.rs

```diff
@@ -46,6 +46,8 @@ pub struct LegacyStockLineRow {
     #[serde(rename = "om_item_variant_id")]
     #[serde(default)]
     pub item_variant_id: Option<String>,
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    pub vvm_status: Option<String>,
```

Comment:
You can rename here if the name is different when deserialising like:
`
    #[serde(rename = "vvm_status")]
    #[serde(deserialize_with = "empty_str_as_option_string")]
    pub vvm_status_id: Option<String>,
`